src/ui/prompts/overridePrompts.ts

```diff
@@ -0,0 +1,72 @@
+import { fetchEnvironments } from '../../api/environments'
+import { fetchFeatures } from '../../api/features'
+import { fetchFeatureOverridesForUser, fetchProjectOverridesForUser } from '../../api/overrides'
+import { Environment, Feature } from '../../api/schemas'
+import { autocompleteSearch } from '../autocomplete'
+import { EnvironmentChoice } from './environmentPrompts'
+import { FeatureChoice } from './featurePrompts'
+
+type Input = {
+    token: string,
+    projectKey: string,
+    featureKey?: string,
+}
+export const overridesFeatureChoices =
+    async (input: Input, search: string): Promise<FeatureChoice[]> => {
+        const features = await fetchFeatures(input.token, input.projectKey)
+        const projectOverrides = await fetchProjectOverridesForUser(input.token, input.projectKey)
+
+        const choices: FeatureChoice[] = []
+        for (const override of projectOverrides) {
+            const feature = features.find((feature: Feature) => feature._id === override._feature)
+            if (!feature) {
+                continue
+            }
+            if (choices.find((choice: FeatureChoice) => choice.value._id === feature._id)) {
+                continue
+            }
+            choices.push({
+                name: feature.name || feature.key,
+                value: feature
+            })
+        }
+        return autocompleteSearch(choices, search)
```

Comment:
I think we need some kind of empty state message if there are no features with overrides (`choices` array is empty) - same case for environments below
<img width="761" alt="image" src="https://github.com/DevCycleHQ/cli/assets/98763537/922688f5-db57-484f-8188-8492f14ca958">
I'm stuck on this step when I have no overrides

Reply:
yess will add!

Reply:
this is the same thing that happens when you create a project with no features and do `feature update` the list is empty.. couldn't find a way to make it not not execute here without making an extra call to the api

Reply:
just escape early, if no results found then return and say `No Overrides found.` with an info print

Reply:
added empty states